Source/WebCore/html/canvas/PlaceholderRenderingContext.cpp

```diff
@@ -61,21 +64,38 @@ void PlaceholderRenderingContextSource::setPlaceholderBuffer(ImageBuffer& imageB
     std::unique_ptr serializedClone = ImageBuffer::sinkIntoSerializedImageBuffer(WTFMove(clone));
     if (!serializedClone)
         return;
-    callOnMainThread([weakPlaceholder = m_placeholder, buffer = WTFMove(serializedClone)] () mutable {
-        RefPtr placeholder = weakPlaceholder.get();
```

Comment:
I suspect you need to keep this as calling m_placeholder->protectedCanvas() seems unsafe (though maybe this file still allows new warnings).